android/src/main/java/com/frontegg/android/embedded/FronteggWebClient.kt

```diff
@@ -118,7 +126,10 @@ class FronteggWebClient(val context: Context) : WebViewClient() {
         }
 
         if (oauthUrls.find { u -> url.toString().startsWith(u) } != null) {
-            return OverrideUrlType.SocialLoginRedirectToBrowser
+            if(url.query?.contains("external=true") == true) {
```

Comment:
you can remove the` == true` from the if

Reply:
query maybe null, it will cause syntax error